plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageSettings.java

```diff
@@ -192,6 +191,31 @@ public void activatePage() {
         //getContainer().updateTitleBar();
     }
 
+    private void createAdditionalRequiredPages() {
```

Comment:
I doubt about calling this method `create..`. I would rather expact to call it like `load..` or smth like that. Yes, you call createControl method, but the control is not visible and so on..